assets/prototype/application/hooks/useZonedTime.ts

```diff
@@ -0,0 +1,21 @@
+import { zonedTimeToUtc, utcToZonedTime } from 'date-fns-tz';
+import useConfig from '../../application/services/config/useConfig';
+
+const useZonedTime = () => {
```

Comment:
The name of the hook is inspired by the `date-fns-tz` function names that are used here.

Reply:
CNO says:

> how about `useTimeZoneTime` ?

only because `useZonedTime` is so awkward to say since `use` ends in a zzzz sound followed immediately by an actual Z, then `Zoned` ends with the plosive `d` sound followed immediately by the plosive `t` sound of Time. Hard to say that without either pausing to get those transitions in properly or having the name come out sounding like 'useOwnTime`

Reply:
Thx, that is a better name. updated :bowing_man: